.github/workflows/integration-test.yaml

```diff
@@ -225,10 +241,40 @@ jobs:
             exit 1
           fi
           ihsConsole=$(echo $outputs | jq -r '.ihsConsole.value')
-          curl $ihsConsole
-          if [[ $? -ne 0 ]]; then
-            echo "Failed to access ${ihsConsole}."
-            exit 1
+          if [[ $selectLoadBalancer == "ihs" ]]; then
+            curl $ihsConsole
+            if [[ $? -ne 0 ]]; then
+              echo "Failed to access ${ihsConsole}."
+              exit 1
+            fi
+          else
+            if [[ $ihsConsole != "N/A" ]]; then
+              echo "ihsConsole is ${ihsConsole}, but it should be N/A as IHS is not enabled."
+              exit 1
+            fi
+          fi
+          appGatewayHttpURL=$(echo $outputs | jq -r '.appGatewayHttpURL.value')
+          appGatewayHttpsURL=$(echo $outputs | jq -r '.appGatewayHttpsURL.value')
+          if [[ $selectLoadBalancer == "appgw" ]]; then
+            curl $appGatewayHttpURL
+            if [[ $? -ne 0 ]]; then
+              echo "Failed to access ${appGatewayHttpURL}."
+              exit 1
+            fi
+            curl $appGatewayHttpsURL -k

```

Comment:
It is probably safer to add a timeout and retry. Because there are multiple curl commands, define the values in the environment section so you can change them easily if desired.

Reply:
Good catch, fixed in [41ccb83](https://github.com/WASdev/azure.websphere-traditional.cluster/pull/193/commits/41ccb8395a12b0dbb856bf052857fc8721a5eb77).